specs/AdaptiveCardsXamarinSpec.md

```diff
@@ -0,0 +1,197 @@
+| Resource Resolvers | ```(NSObject<ACOIResourceResolver>) getResourceResolverForScheme: (NSString*)``` | P2 |
+| Feature Registration | ```(void) addFeature: (NSString*) featureVersion: (NSString*)``` | P0 | 
+| Feature Registration | ```(NSString*) removeFeature: (NSString*)``` | P0 |
+| Feature Registration | ```(NSString*) getFeatureVersion: (NSString*)``` | P0 |
+| Resource Information | ```(NSArray<ACORemoteResourceInformation>*) remoteResourceInformation``` | P2 | 
+
+## Breaking changes
+
+There is one big breaking change to be introduced in the iOS platform due to the Xamarin bindings having a direct relationship with the name of the library, which requires changing the library name from ACFramework to AdaptiveCards.
+
+Besides that, no breaking changes should be introduced as the native platforms' code is only consumed and projected into C#.
+
+## Test Plan
+
+For the alpha and pre-release versions the testing will be performed manually
+
+## Release plan
+
+### Artifacts
+
+The artifacts to be released with this library will be:
+1. Source code for Xamarin bindings in the github repository
+1. Xamarin.Android nuget package
+1. Xamarin.iOS nuget package
+
+### Continuous Integration and Release
+
+To be able to release the previously mentioned nuget packages build definitions must be created. The current system we use for building the other packages allow us to build Xamarin.Android and Xamarin.iOS packages
+
+#### Pre-requisites
+
+The Xamarin packages require native libraries to be built beforehand, as we can't upload binaries into the repository, then the Android and iOS build definitions must be updated to copy the generated binaries into the xamarin projects folder. 
+
+#### Build definitions
+
+At least four build pipelines will be needed for this project, 2 pipelines for Continuous integration and 2 pipelines for the release. The pipelines for the release will be similar to the UWP and WPF pipelines that generate nuget packages and build against a visual studio solution/project.
+
+## Remaining Tasks
+
+For the alpha version all P0 and P1 APIs should be done while P2 and P3 tasks must be done for the first complete release. 
+
+The priorities among the APIs to be ported into Xamarin for the pre-release version were decided by selecting which APIs would be the bare minimum for a consumer of the Xamarin package to be able to render a card would be P0 as well as any APIs that were part of a feature (such as featureRegistration). For P1, the tasks were decided using the same features asked for P1 in the [Xamarin.Forms github issue](https://github.com/microsoft/AdaptiveCards/issues/3255) which states that custom rendering for actions and elements would be P1. Finally, P2 would be any task that gives a better developer experience but are not required for using Adaptive Cards and P3 would be any APIs that may not be used by most consumers or are specific to only one platform.
+
+## Open issues
```

Comment:
Another one i just remembered is what is the Accessibility story here for hte library? Are there expectations/guidance for Xamarin bindings authors at all ? Are there tools we can leverage? @mikehill thoughts?